backend/app/models/cif_meta.py

```diff
@@ -119,7 +119,7 @@
         },
         "PWS_Violations_Since_2016": {
             "label": "Public Water System Violations since 2016",
-            "unit": MeasureUnit.PERCENT,
+            "unit": MeasureUnit.COUNT,
```

Comment:
This seems to be an absolute count. Unless there is a division missing somewhere in the ingest.

Reply:
No, you're right; it's supposed to be a count. Nice catch!